Example/Example/EditorDemoController.swift

```diff
@@ -76,17 +76,17 @@ class EditorDemoController: UIViewController {
         return textView
     }()
 
-    fileprivate(set) lazy var titleTextField: UITextView = {        
-        let textField = UITextView()
+    fileprivate(set) lazy var titleTextField: UITextField = {
+        let textField = UITextField()
 
         textField.accessibilityLabel = NSLocalizedString("Title", comment: "Post title")
         textField.delegate = self
         textField.font = UIFont.preferredFont(forTextStyle: UIFontTextStyle.headline)
         textField.returnKeyType = .next
         textField.textColor = UIColor.darkText
         textField.translatesAutoresizingMaskIntoConstraints = false
-        textField.isScrollEnabled = false
         textField.backgroundColor = .clear
+        textField.addTarget(self, action: #selector(textFieldDidChange(_ :)), for: .editingChanged)
```

Comment:
This line is causing the build to fail on Travis: https://travis-ci.org/wordpress-mobile/AztecEditor-iOS/builds/314557155?utm_source=github_status&utm_medium=notification

I think the issue here is that your fork seems to be behind the main AztecEditor-iOS repo — we've upgraded to Swift 4 in the meantime, which has different rules about interacting with Objective-C code.